### ISSUE TITLE
Canvas BG Color

### ISSUE THREAD

**Issue by deathchimera** [opentoonz/opentoonz#5268]
### Is your feature request related to a problem? Please describe

Mostly just a nitpick, but it's a bit disorienting only having the camera background opaque while the rest of the canvas is transparent. You can fill or use template images, but it feels like something better handled programmatically by OpenToonz. I run into this the most when playing with odd canvas sizes for big camera movements.

### Describe the solution you'd like

Basically "View > Camera BG Color" and it's implementation, but for the canvas. I can't think of any negative edge cases that arise from this, and it would be a toggle option anyways. I assume "Camera BG Color" would always render over "Canvas BG Color", but that can be an option too if need be.

**Comment by RodneyBaker** [comment]
I apologize @deathchimera, when you posted this feature request I didn't quite grasp what you were after relative to what options we have now.  That is still the case.

Perhaps if you can add some more information that will lead to the feature you request being added.

**Comment by RodneyBaker** [comment]
Still waiting on clarifying information on this request.

**Comment by deathchimera** [comment]
Sorry for the late reply. I was trying to say that I wanted a viewer coloring option for the space between the Camera Box and Raster Bounding Box. It can be hard to see dark lines on dark themes without those premade safe-area templates that professionals often use.

![Image](https://github.com/user-attachments/assets/3072afec-f8a7-4eae-8b73-97e742396e7e)

![Image](https://github.com/user-attachments/assets/cb7d5a73-319c-426f-a014-ef17f4da11dd)

I'm essentially just looking for this effect  (ignore random Azumanga, got image from a stream)

![Image](https://github.com/user-attachments/assets/8d43d31f-72a6-436f-9833-bf751cc77143)

**Comment by RodneyBaker** [comment]
Just when I think I'm understanding... I realize I'm not.
In what way do the current color changing options of Preferences and Scene Settings not meet your requirement?

In the attached video is it the areas where I place the Xs the specific area you want to be a different color?

https://github.com/user-attachments/assets/e49f5204-e17c-42c7-8da4-0a54f0be6383

I'm realizing even as I post this that I should have scaled the imported image to adjust the raster bounding box in relationship to the camera box/canvas/table

**Comment by deathchimera** [comment]
Yes, the areas you mark are the ones I'm referring to. Realistically it would probably be set to be the same value as Camera BG Color, or a shade in difference, but ideally the user could just set it to whatever color they want in the "Preferences>Colors" tab as you show in the video. Implementation might include a "Canvas BG Color" toggle option under the View menu.
